### PR TITLE
Added `export-schema` command to Docs

### PR DESCRIPTION
**Pull request by Ckk3**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Documentation

## Issues Fixed or Closed by This PR

* #463 

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the CONTRIBUTING document.
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage)

## Additional Information

I've also added the new page in the Guide menu: 

image

## Summary by Sourcery

This pull request adds new documentation for the `export_schema` management command, explaining its purpose, usage, and providing an example. The new documentation page has been added to the Guide section in the navigation menu.

* **Documentation**:
    - Added documentation for the `export_schema` management command, detailing its usage, arguments, and options.
    - Included an example of how to use the `export_schema` command to export a GraphQL schema.

### CODE REVIEW

File: docs/guide/export-schema.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+python manage.py export_schema <schema_location> --path <output_path>
+```
+
+### Arguments
+
+- `<schema_location>`: The location of the schema module. This should be a dot-separated Python path (e.g., myapp.schema).
+
+### Options
+
+- `--path <output_path>`: An optional argument specifying the file path where the schema should be saved. If not provided, the schema will be printed to standard output.
```
[the remark this answers is not in the document]
Comment (Ckk3) [reply docs/guide/export-schema.md:23]: I think this is reduntant so I'll not add.

File: docs/guide/export-schema.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+To use the `export_schema` command, you need to specify the location of the schema. Optionally, you can provide a file path to save the schema. If no path is provided, the schema will be printed to the console.
+
+```sh
+python manage.py export_schema <schema_location> --path <output_path>
+```
+
+### Arguments
+
+- `<schema_location>`: The location of the schema module. This should be a dot-separated Python path (e.g., myapp.schema).
```
[the remark this answers is not in the document]
Comment (Ckk3) [reply docs/guide/export-schema.md]: Added!

File: docs/guide/export-schema.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+!!! info
+
+    The `export_schema` management command provided here is specifically designed for use with `strawberry_django`. The [default Strawberry export command](https://strawberry.rocks/docs/guides/schema-export) won't work with `strawberry_django` schemas, so you should use this command to ensure proper schema export functionality.
+
+The `export_schema` management command allows you to export a GraphQL schema defined using the `strawberry_django` library. This command converts the schema definition to a text format, which can then be saved to a file or printed to the console.
+
+## Usage
+
+To use the `export_schema` command, you need to specify the location of the schema. Optionally, you can provide a file path to save the schema. If no path is provided, the schema will be printed to the console.
```
[the remark this answers is not in the document]
Comment (Ckk3) [reply docs/guide/export-schema.md]: Added!

File: docs/guide/export-schema.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# Export Schema
+
+!!! info
+
+    The `export_schema` management command provided here is specifically designed for use with `strawberry_django`. The [default Strawberry export command](https://strawberry.rocks/docs/guides/schema-export) won't work with `strawberry_django` schemas, so you should use this command to ensure proper schema export functionality.
+
+The `export_schema` management command allows you to export a GraphQL schema defined using the `strawberry_django` library. This command converts the schema definition to a text format, which can then be saved to a file or printed to the console.
```
[the remark this answers is not in the document]
Comment (Ckk3) [reply docs/guide/export-schema.md]: Added!

File: docs/guide/export-schema.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
+# Export Schema
+
+!!! info
+
+    The `export_schema` management command provided here is specifically designed for use with `strawberry_django`. The [default Strawberry export command](https://strawberry.rocks/docs/guides/schema-export) won't work with `strawberry_django` schemas, so you should use this command to ensure proper schema export functionality.
```
[the remark this answers is not in the document]
Comment (Ckk3) [reply docs/guide/export-schema.md]: Added!

**Comment by Ckk3** [comment]
Please let me know if this PR need some changes!

File: docs/guide/export-schema.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bellini666) [reply docs/guide/export-schema.md:23]: I agree that it is redundant hahaha

**Review by bellini666 — APPROVED** [review]

**Comment by bellini666** [comment]
Awesome! Thanks for this :)

**Pull request merged**